.github/workflows/validate-pull-request.yml

```diff
@@ -0,0 +1,9 @@
+name: Validate Pull Request
+
+on:
+  pull_request:
+
+jobs:
+  validate_pull_request:
+    name: Call validate workflow
```

Comment:
```suggestion
```

Reply:
done

Reply:
This is still here, and the job name is still the same.